ext/tidy/config.m4

```diff
@@ -49,11 +56,17 @@ if test "$PHP_TIDY" != "no"; then
     ], [], [])
   ],[])
 
+  PHP_CHECK_LIBRARY($TIDY_LIB_NAME,tidyReleaseDate,
```

Comment:
For curiosity, which version doesn't have it ?

Reply:
tested with tidyp 1.0.4